src/components/more-blogs-section/more-blogs-section.jsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+
+import { Link } from "gatsby";
+import PropTypes from "prop-types";
+
+import { classnames } from "../../helpers";
+
+import { Card } from "../card";
+
+import * as styles from './more-blogs-section.module.scss';
+
+const MoreBlogsSection = ({ data, refMoreFrom, threeBlogsSectionTitle }) => {
```

Comment:
```suggestion
const MoreBlogsSection = ({ data, refMoreFrom, title }) => {
```

Reply:
Also is missing define this propr in propTypes

Reply:
done